specification/loadtestservice/LoadTestService/routes.tsp

```diff
@@ -33,35 +44,11 @@ numeric, underscore or hyphen characters.
 
   @summary("Delete a test by its name.")
   @doc("Delete a test by its name.")
-  @route("/tests/{testId}")
-  @delete
-  DeleteTest is Azure.Core.Foundations.Operation<
-    {
-      @doc("""
-Unique name for the load test, must contain only lower-case alphabetic,
-numeric, underscore or hyphen characters.
-""")
-      @path
-      testId: string;
-    },
-    void
-  >;
+  deleteTest is StandardResourceOperations.ResourceDelete<Test>;
 
   @summary("Get load test details by test name")
   @doc("Get load test details by test name")
```

Comment:
Get load test details by testId?

Reply:
Correcting this but these descriptions were taken from GA swagger.